scripts/stress-app.js

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+'use strict';
+
+/* eslint-disable no-console */
+const exec = require('child_process').exec;
+const temp = require('temp');
+
+const outputPath = temp.mkdirSync('angular-stress-tester');
+
+function execute(command) {
+  return new Promise((resolve, reject) => {
+    exec(command, (error, stdout) => {
+      if(error) {
+        reject(error);
+      }
+      resolve(stdout.trim())
+    });
+  });
+}
+
+Promise.resolve()
+  .then(() => console.log('Cloning application...'))
+  .then(() => execute(`git clone "https://github.com/sumitarora/angular-stress-tester" "${outputPath}"`))
+  .then(() => console.log('Moving into Directory...'))
+  .then(() => process.chdir(outputPath))
+  .then(() => console.log(`Current Directory: ${process.cwd()}`))
+  .then(() => console.log('Installing...'))
+  .then(() => execute('npm install'))
+  .then(() => console.log('Building...'))
```

Comment:
The goal of such a big app is to time the build step, so maybe insert some performance metric gathering here and show it at the end? We really don't care about the previous steps' performance.